### PR TITLE
[Doc] add corr related functions and update window functions

### PR DESCRIPTION
**Pull request by evelynzhaojie**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/sql-reference/sql-functions/date-time-functions/add_months.md [hunk 1]
```diff
@@ -2,24 +2,26 @@
...
 ## Description
 
-Adds an integer months to a given date (DATE, DATETIME). The integer can be positive or negative.
+Adds a specified number of months to a given date (DATE or DATETIME). The [months_add](./months_add.md) function provides similar functionalities.
 
-The resulting day component remains the same as that specified in `date`, unless the resulting month
-has fewer days than the day component of the given date, in which case the day will be the last day of
-the resulting month.
+The day component in the resulting month remains the same as that specified in `date`, unless the resulting month has fewer days than the day component of the given date, in which case the day will be the last day of
+the resulting month. For example, `select add_months('2022-01-31', 1);` returns `2022-02-28 00:00:00`. If the resulting month
```
Comment (EsoragotoSpirit) [inline docs/sql-reference/sql-functions/date-time-functions/add_months.md:8]: 取消换行

File: docs/sql-reference/sql-functions/date-time-functions/add_months.md [hunk 2]
```diff
@@ -2,24 +2,26 @@
...
+has more days than the day component of the given date, the result has the same day component as `date`. For example, `select add_months('2022-02-28', 1)` returns `2022-03-28 00:00:00`.
```
Comment (EsoragotoSpirit) [inline docs/sql-reference/sql-functions/date-time-functions/add_months.md:9]: 取消换行

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**